<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kindle VocabMate</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Kindle vocab.db File Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dbPathEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="dbPathOpenButton">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadDBButton">
        <property name="text">
         <string>Load DB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Select Book:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="bookListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="iconSize">
           <size>
            <width>51</width>
            <height>78</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>After Timestamp</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="timestampEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>1</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="timestampDateButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Date...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="loadVocabButton">
          <property name="text">
           <string>Load Vocabulary</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="vocabTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Dictionary Language:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="langComboBox"/>
            </item>
            <item>
             <widget class="QListWidget" name="dictionaryListWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="iconSize">
               <size>
                <width>51</width>
                <height>78</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="addDefinitionButton">
            <property name="text">
             <string>Add Definitions</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Output File:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outputFileEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="outputFileSelectButton">
            <property name="text">
             <string>Select...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="saveTSVButton">
          <property name="text">
           <string>Save TSV File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKindle_DB">
    <property name="title">
     <string>Kindle DB</string>
    </property>
    <addaction name="actionOpen_Vocab_DB"/>
    <addaction name="actionLoad_DB"/>
   </widget>
   <addaction name="menuKindle_DB"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Vocab_DB">
   <property name="text">
    <string>Open Vocab DB...</string>
   </property>
  </action>
  <action name="actionLoad_DB">
   <property name="text">
    <string>Load DB</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
